content/docs/reference/pytorchjob/v1beta1/pytorch.md

```diff
@@ -0,0 +1,226 @@
++++
```

Comment:
Please would you also add a file at the following location:
content/docs/reference/pytorchjob/v1beta1/_index.md

With file content similar to this one:
https://github.com/kubeflow/website/blob/master/content/docs/reference/tfjob/v1beta1/_index.md

And similar for:
content/docs/reference/pytorchjob/v1beta2/_index.md

These index files will sort out the navigation structure in the left-hand menu of the doc site.